prow/cluster/500-kubernetes-external-secrets_crd.yaml

```diff
@@ -1,236 +0,0 @@
----
```

Comment:
What's the reason for deleting/removing these files?

Reply:
Kubernetes External Secrets has been deprecated in favour of External Secrets Operator. 

Config for ESO is split across crd.yaml and deploy.yaml

Reply:
Cool, thanks for updating it!